ui/src/criteria/unhintedValue.tsx

```diff
@@ -105,36 +105,49 @@ class _ implements CriteriaPlugin<string> {
     };
   }
 
-  generateFilter() {
+  generateFilter(occurrenceId: string) {
     const decodedData = decodeData(this.data);
 
     if (this.config.groupByCount) {
       return null;
     }
 
-    return {
-      type: FilterType.Attribute,
-      attribute: this.config.attribute,
-      ranges: [rangeFromData(decodedData)],
-    };
+    const attributes = this.config.attributes[occurrenceId];
+    if (!attributes || !attributes.values.length) {
+      throw new Error(`No attributes found for occurrence: ${occurrenceId}`);
```

Comment:
Does this disallow no group by attributes? If so, I think we do want to accommodate that use case (e.g. has at least 2 height measurements) and the backend should already allow that.

Reply:
Yes. Changed to allow empty attribute lists.

Reply:
OK -- was this change elsewhere? I don't think this particular section changed at all.

Reply:
Yeah, the group by version is below. This could potentially be removed at some point since we're not using it anymore.